docs/CHANGELOG.rst

```diff
@@ -10,6 +10,10 @@ This project adheres to `Semantic Versioning <http://semver.org/>`_.
 Unreleased
 ==========
 
+Added
+-----
+- Add unaccented full text search configuration ``simple_unaccent``
```

Comment:
This line feels unnecessary. Include it in the line below.

Reply:
I have merged commits that create and use this configuration.